packages/common/src/ConfigResponse.ts

```diff
@@ -105,6 +106,16 @@ export interface GSNConfig {
    */
   paymasterAddress?: Address
 
+  /**
```

Comment:
I really dislike the fact we need to add here token-specific values.
it means our extension mechanism to add paymasters is broken. 
see the suggestion here: https://hackmd.io/vBD0nJKZRda3Ns-5brQ4qg

Reply:
Again, this is because we don't currently have a way to update clients code if we add/remove a token. This is not the case for all paymasters as most are written by the dapp devs who can also update the client code. This is exactly the mechanism that we added to dynamically update the client config.